docs/source/Support/bskReleaseNotes.rst

```diff
@@ -47,6 +47,7 @@ Version |release|
   upgrading from previous versions must update their scripts to store these objects on their
   simulation instance to prevent segmentation faults. Once again, this change replaces the previous use of
   ``.disown()`` with a more robust memory management approach.
+- Added capability for defining Locations using latitude/longitude/altitude
```

Comment:
be more specific.  Say you adding method XXX to the `vizSupport.py` script and make the script a hyperlink to the HTML support page.